hilfe/handbuch/probleme/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: layouts/hilfe/single.njk
+title: Probleme
+weight: 6
+icon: exclamation-triangle
+---
+
+    **"Irren ist menschlich" - und die Fülle gesetzlicher Bestimmungen enorm.**
+
+So gern wir auch auf Anhieb den perfekten Dienstplan erstellen oder alle
+Schichten im Blick haben würden - Fehler schleichen sich immer ein. Besonders
+wenn man die diversen **Arbeitsrichtlinien und Vertragsvereinbarungen** dabei im
+Blick behalten muss. Die **Problemberichte** in Pentacode helfen Ihnen dabei einen
+klaren Überblick zu behalten ob Dienstplan, Schichten und Kassenbuch reibungslos
+laufen - und **Probleme zu vermeiden bevor sie überhaupt entstehen**. Das spart
+Ihnen nicht nur Zeit, sondern auch Nerven bei der nächsten Buchprüfung, da Sie
+sicher sein können, stets im Recht gehandelt zu haben. 
+
+## Verschiedene Arten von Problemen
+
+Pentacode zeigt Ihnen in der Problemübersicht **acht Arten von
+Problemberichten** an. Um diese möglichst schnell beheben zu können, sollten Sie
+wissen wo der Ursprung jedes dieser Probleme liegt. Wie unter [Problem
+beheben](#problem-beheben) beschrieben, bringt Sie der {%button label="Beheben",
+icon="pencil"%} immer direkt zum Kern des Problems.  
+
+### Probleme aus der Dienstplanung
+
+Diese Problemberichte warnen Sie vor Problemen in der Dienstplanung, damit Sie
+diese präventiv beheben können.
```

Comment:
Finde ich gut, dass du versucht hast, eine logische Unterteilung zu finden. Würde ich aber weglassen, da die Zuordnung nicht immer eindeutig ist (Arbeitszeit > 10 Stunden kann z.B. auch bei abgeschlossenen Schichten auftreten).

Reply:
Ich hab das jetzt noch nicht geändert weil ich eigentlich schon finde, dass das Sinn macht - es geht ja um den Informationsgehalt wo das Problem zu lösen ist. Und der ist bei z.B Arbeitszeit >10 Stunden meist in der Planung - in zurückliegenden Schichten entsprang das Problem dann eben auch einer unpassenden Planung. Für die Fälle, dass Arbeitszeit <10h und Ruhezeit<11h durch Überstunden ausgelöst wird hab ich noch einen Disclaimer geschrieben.

Ich würd also eher die Einteilung so lassen - geht ja eher um Orientierung.

Reply:
Weiß dein Feedback zu schätzen, aber bitte trotzdem rausnehmen 😉